app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -300,13 +337,31 @@ class NewMessageFragment : Fragment() {
         }
     }
 
+    private fun initEditorUi() = with(binding) {
+        editor.apply {
+            enableAlgorithmicDarkening(isEnabled = true)
+            if (context.isNightModeEnabled()) editor.addCss(context.readRawResource(R.raw.custom_dark_mode))
+
+            addCss(context.readRawResource(R.raw.style))
+            addCss(context.readRawResource(R.raw.editor_style))
+
+            val isPlaceholderVisible = combine(
+                isEmptyFlow.filterNotNull(),
+                newMessageViewModel.isShimmering,
+            ) { isEditorEmpty, isShimmering -> isEditorEmpty && !isShimmering }
+
+            isPlaceholderVisible
+                .onEach { isVisible -> newMessagePlaceholder.isVisible = isVisible }
+                .launchIn(lifecycleScope)
+        }
+    }
```

Comment:
I tried doing this but I didn't find any way of doing it easily like you did. Apparently I wasn't looking at it the correct way because it wasn't even that hard to simplify